app/src/main/res/layout/fragment_map.xml

```diff
@@ -13,11 +14,15 @@
 
     <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
         android:id="@+id/slide_menu"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
+        android:layout_width="408dp"
```

Comment:
why not match_parent?

Reply:
or i guess if you're using the constraints you can set the height and width to 0